src/screens/nano/NanoContractDetail.js

```diff
@@ -53,7 +55,10 @@ function NanoContractDetail(props) {
   const [errorMessage, setErrorMessage] = useState('');
 
   useEffect(() => {
-    loadData();
+    if (nc) {
+      // The nano contract has just been unregistered
+      loadData();
+    }
   }, []);
```

Comment:
The nc exists when the contract is unregistered?
Is it not the opposite?

Reply:
Actually this comment was not good. I had to add this conditional because this was throwing an error when the nano contract had been unregistered, so the `loadData` method was failing.

The best comment would have been `Load data only if nano contract exists in redux, otherwise it has just been unregistered.`